client/components/Menubar/MenubarSubmenu.tsx

```diff
@@ -18,7 +17,17 @@ import {
 } from './contexts';
 import TriangleIcon from '../../images/down-filled-triangle.svg';
 
-export function useMenuProps(id) {
+/** Custom subset of valid list item roles for the Menubar list items */
+export enum MenubarListItemRole {
```

Comment:
small nit but it looks like these are values for the container, not the items within the container? should we update the comment and interface name?

Reply:
maybe like "MenuContainerRole"